### ISSUE TITLE
VS 2019?

### ISSUE THREAD

**Issue by ColbyRingeisen** [MicrosoftDocs/visualstudio-docs#3949]
This sample makes no sense in Visual Studio 2019.  Please update!

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 61c068da-293e-ea09-c3ed-be7d3c7da17d
* Version Independent ID: 70af6750-b737-f536-8ebc-ad3a24becb80
* Content: [Build an OpenGL ES Application on Android and iOS - Visual Studio](https://docs.microsoft.com/en-us/visualstudio/cross-platform/build-an-opengl-es-application-on-android-and-ios?view=vs-2019)
* Content Source: [docs/cross-platform/build-an-opengl-es-application-on-android-and-ios.md](https://github.com/MicrosoftDocs/visualstudio-docs/blob/master/docs/cross-platform/build-an-opengl-es-application-on-android-and-ios.md)
* Product: **visual-studio-windows**
* Technology: **vs-ide-mobile**
* GitHub Login: @corob-msft
* Microsoft Alias: **corob**

**Comment by corob-msft** [comment]
@ColbyRingeisen 

On the assumption that you were blocked by the new project creation process, I've updated the steps for VS 2019 for publication tomorrow. The remaining steps worked the same for me for the Android app. Make sure the Android emulator is set up and runs independently before you launch it from Visual Studio. I don't have a Mac to test the iOS build with right now, but if there are problems, raise another issue on this topic and I'll look into it.
